torch/_higher_order_ops/cond.py

```diff
@@ -471,6 +471,15 @@ def _bound(s0, lower_bound: bool):
         mode.shape_env.constrain_symbol_range(merged_out.node.expr, *min_max(a, b))
         return merged_out
 
+    # FakeTensorMode dispatch is before tensor subclass dispatch
```

Comment:
cc @bdhirsh this is the special casing I mentioned to you. Need to change the comment